src/components/ArmyBuildPane.jsx

```diff
@@ -0,0 +1,23 @@
+
+import React from 'react';
+import styled from 'styled-components'
+
+import ArmyList from'./ArmyList';
+import SelectionPane from './SelectionPane'
+
+const Container = styled.div`
```

Comment:
Consider if these wrapper components names can be more expressive. Sometimes they can't but sometimes they can! Maybe this could be just called `Flex` or `HorizontalFlex`?

Reply:
👌

Reply:
Thanks for these thoughts guys! Lots of good points, mostly these are place holders for now, but hopefully better names will emerge as they evolve and get replaced haha